src/lib/plugins/clear.js

```diff
@@ -0,0 +1,204 @@
+module.exports.server = (serv, { version }) => {
+  const mcData = require('minecraft-data')(version)
+  const ChatMessage = require('prismarine-chat')(version)
+  const notUdf = x => x !== undefined
```

Comment:
Why define a function for this ? Just inline it.

Reply:
done